LICENSE

```diff
@@ -0,0 +1,15 @@
+ISC License
```

Comment:
Not sure if we have a company-wide policy we follow here but usually new stuff we do uses `Apache-2`.

Reply:
Ok, I confirm with @ganeshvlrk on that

Reply:
@ganeshvlrk suggested we use MIT, to be consistent with other projects.

I've switched it to MIT then